response_operations_ui/templates/change-response-status.html

```diff
@@ -0,0 +1,106 @@
+{% extends "layouts/base.html" %}
```

Comment:
In general this html seems a bit spaced out, might be good to make it a little bit more compact

Reply:
Yeah sure it was just a lazy code and paste :)